Zend/zend_compile.c

```diff
@@ -9135,7 +9302,14 @@ void zend_compile_expr(znode *result, zend_ast *ast) /* {{{ */
 }
 /* }}} */
 
-zend_op *zend_compile_var(znode *result, zend_ast *ast, uint32_t type, int by_ref) /* {{{ */
+void zend_compile_expr(znode *result, zend_ast *ast)
+{
+	uint32_t begin_short_circuiting_chain_flags = zend_begin_short_circuiting_chain(ast->kind, 0);
+	zend_compile_expr_inner(result, ast);
+	zend_end_short_circuiting_chain(begin_short_circuiting_chain_flags, result);
+}
+
+zend_op *zend_compile_var_inner(znode *result, zend_ast *ast, uint32_t type, int by_ref)
```

Comment:
static

Reply:
Also wasn't sure about naming here, `_inner` kind of sucks but couldn't think of anything better.